webapp/views/forgotpass.py

```diff
@@ -0,0 +1,36 @@
+from django.views import View
+from django.shortcuts import render, redirect
+from django.http import HttpResponse
+from webapp.models import User, Verification
+from secrets import token_urlsafe
+from django.contrib.sites.shortcuts import get_current_site
+from django.core.mail import send_mail
+import random as r
+
+otp=""
+for i in range(6):
+    otp+=str(r.randint(1,9))
```

Comment:
from random import randint
otp = randint(100000,999999)

Would be more efficient, eliminating the use of a for loop.